HW-2/src/main/java/org/example/Main.java

```diff
@@ -1,7 +1,11 @@
 package org.example;
 
 public class Main {
-    public static void main(String[] args) {
+    public static void main(String[] args)  {
 
+        ValueCalculator calculator = new ValueCalculator(1);
+        calculator.executeWithTwoThreads();
```

Comment:
Please, use one method with input parameter as an amount of threads

Reply:
The method executeWithTwoThreads() was deleted